tests/test_core.py

```diff
@@ -229,7 +250,7 @@ def __test(ref_power, clip):
             if six.callable(ref_power) or ref_power >= 0.0:
                 tf = __test
             else:
-                tf = raises(librosa.ParameterError)(__test)
+                tf = raises(ValueError)(__test)
```

Comment:
Hi Brian,
sorry about that, I commited the same files that I first commited in the master branch without taking care of these differences.
I just reset and redid the commits.